roles/openshift_master_upgrade/meta/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  author: your name
+  description: OpenShift Master upgrade
+  company: Red Hat, Inc.
+  license: Apache License, Version 2.0
+  min_ansible_version: 2.1
+  platforms:
+  - name: EL
+    versions:
+    - 7
+dependencies:
+- role: openshift_common
```

Comment:
So, within the context of play, you should not see a role listed as a dependency being run more than once, unless it is being called explicitly with different variables. There is a bug in Ansible that we would need to track down if that is the case.

If we are going to add requirements for implicit dependencies rather than explicit, then the role docs should make that clear. In this case, calling the needed steps out in the Requirements section rather than just relying on the example would probably be good.